scripts/fetch_templates.py

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env python
+
+"Pre-emptive caching of commonly used TemplateFlow templates"
+import templateflow.api as tf
+
+
+def fetch_MNI2009(validate=True):
+    # T1w
+    tf.get(
+        'MNI152NLin2009cAsym',
+        atlas=None,
+        resolution=(1, 2),
+        extension='.nii.gz'
```

Comment:
I think we can get rid of the extension selector, if you look at the just merged #2268 changes.